Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.provision "shell", path: "bootstrap.sh"
+
+  config.vm.provider "virtualbox" do |v|
+      v.memory = 2048
+  end
+
+  config.vm.synced_folder ".", "/vagrant"
```

Comment:
This comes for free, so we don't need to add a configuration item for it unless we want to explicitly override an option or disable the share.